umpleonline/counter.php

```diff
@@ -35,8 +47,18 @@
 	$log = fopen($commandlogpath, "r");
 	$count = fgets($log,1000);
 	fclose($log);
-	$count_new=$count/1000;
-	echo " | $count_new K commands run since July 2019" ;
+	if ($count>=1000 && $count<1000000) {
+		$count_new=$count/1000;
+		echo " | $count_new K commands run since July 2019" ;
+	}
+	elseif ($count>=1000000) {
+		$count_new=$count/1000000;
+		echo " | $count_new million commands run since July 2019" ;
```

Comment:
This does not round to two decimal places as I had asked. So for example if the log file  was 123456789013 the output is 123456.789013, when it should be 123456.78

Same problem above for countlog.txt for the number of visits.